doc/main/sections/experiments.tex

```diff
@@ -0,0 +1,47 @@
+\section{Eksperymenty}
+W naszym projekcie przeprowadziliśmy walidacji krzyżowej dla parametru k równego 10 dla drzewa utworzonego przez algorytm ID3 oraz dla drzewa utworzonego prze C4.5.
+
+\subsection{Wyniki}
+
+
+
+\begin{table}[H]
+    \centering
+    \begin{tabular}{|c|c|}
+    \hline
+    Algorytm                & Dopasowanie         \\ \hline
+    ID3                     & 81,5\%              \\ \hline
+    C4.5                    & 81,4\%              \\ \hline
+    \end{tabular}
+    \caption{Wyniki walidacji krzyżowej dla akceptorów}
+    \label{tab:crossing}
+\end{table}
+
+\begin{table}[H]
+    \centering
+    \begin{tabular}{|c|c|}
+    \hline
+    Algorytm                & Dopasowanie         \\ \hline
+    ID3                     & 83,2\%              \\ \hline
+    C4.5                    & 81,7\%              \\ \hline
+    \end{tabular}
+    \caption{Wyniki walidacji krzyżowej dla donorów}
+    \label{tab:crossing}
+\end{table}
+
+Z wyników walidacji...
```

Comment:
Dla obu przypadków wyniki są dziwne dosyć bo teoretycznie C4.5 jest lepszy... dodatkowo możesz napisać że to średnie dopasowanie bo to chyba to

Reply:
nie wydaje mi się, że powinno być lepsze. Powinno być gorsze, bo drzewo jest przycięte i wartości na liściach są przybliżone. Więc powinny moim zdaniem gorsze, ale algorytm dla C4.5 powinien szybciej działać.

Reply:
ale unika nadmiernego dopasowania, chyba że tu to zjawisko nie występuje dlatego id3 lepszy - może warto to napisać we wnioskach